_includes/scripts.html

```diff
@@ -39,4 +36,64 @@
 </script>
 <!-- Intercom -->
 <script>(function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/rzcjacgd';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);}if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})()</script>
+<script>
+  (function ($) {
+    var cookieInnerHtml = '<div><p>By using this website you agree to our <a href="/cookie-policy/">cookie policy</a></p><button id="cookieModalClose" class="btn btn-primary">OK</button></div>';
+
+    var getCookiebyName = function (name){
+      var pair = document.cookie.match(new RegExp(name + '=([^;]+)'));
+      return !!pair ? pair[1] : null;
```

Comment:
Check `pair.length >= 2`?

Reply:
Same as above, I took the already existing cookie script and moved it to cookie.js

Reply:
Same here. It looks like a potential bug. Just because it was there before doesn't mean we should leave the bug in place!

Reply:
@brikis98 We're passing the name of the cookie we need to check for through the call below, so the first occurrence of the cookie name present validates our check to show or not show the modal.

Reply:
Not sure I follow. `match` will return `null` or an array. You're checking for `null` but not checking the array length before returning `pair[1]`. Yes, per the current regex design, the array _should_ have a length of 2, but that's the thing about bugs: it's possible that it won't, especially as this code is modified over time. Checking that `pair.length >= 2`, in addition to the fact that `pair` is not `null` seems worthwhile to avoid out of bounds errors, right?

Reply:
@sahilakos This looks good now.